packages/paste-core/components/topbar/src/TopbarProductSwitcher/TopbarProductSwitcher.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import {Menu} from '@twilio-paste/menu';
+import type {MenuProps} from '@twilio-paste/menu';
+
+export interface TopbarProductSwitcherProps extends MenuProps {
+  children: NonNullable<React.ReactNode>;
+}
+
+const TopbarProductSwitcher = React.forwardRef<HTMLDivElement, TopbarProductSwitcherProps>(
```

Comment:
question: same question as above, will we ever only want the product switcher to be within Topbar?